src/extension/utilities/utilityFunctions.ts

```diff
@@ -58,6 +79,46 @@ export function importSort(i1: Import, i2: Import, order: 'asc' | 'desc' = 'asc'
 }
 
 /**
+ * Order imports by first specifier name. Does not re-sort specifiers internally:
+ * assumes they were sorted AOT (which happens in `ImportManager#organizeImports`,
+ * indeed).
+ *
+ * @export
+ * @param {Import} i1
+ * @param {Import} i2
+ * @param {('asc' | 'desc')} [order='asc']
+ * @returns {number}
+ */
+export function importSortByFirstSpecifier(i1: Import, i2: Import, order: 'asc' | 'desc' = 'asc'): number {
+    const strA = getImportFirstSpecifier(i1);
+    const strB = getImportFirstSpecifier(i2);
+
+    return localeStringSort(strA, strB, order);
+
+    function getImportFirstSpecifier(imp: Import): string {
```

Comment:
Please add this function as a non exported function (with jsdocs) in the root of the file.

This function would be defined each time you call it, since it does return the correct specifier and is not bound to a closure, you can specify it in the root (top or bottom does not matter.. I'd use the top, beneath the imports)

Reply:
Dang, that's obvious, should have caught on that. Hang on a sec.